runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/testing/TestDataflowRunnerTest.java

```diff
@@ -132,8 +140,8 @@ public void testRunBatchJobThatSucceeds() throws Exception {
     when(mockRunner.run(any(Pipeline.class))).thenReturn(mockJob);
 
     TestDataflowRunner runner = (TestDataflowRunner) p.getRunner();
-    when(request.execute()).thenReturn(
-        generateMockMetricResponse(true /* success */, true /* tentative */));
+    when(request.execute()).thenReturn(generateMockMetricResponse(true /* success */,
```

Comment:
I refactor generateMockMetricResponse() in order to not only generate PAsssert metrics but also watermark metrics if people want. The reason I use null at the third argument is either because batch pipeline nor PAssert will fail so that watermark metrics is not necessary. 

Maybe having another method with same name but only first two arguments will be more clear.